pmlc/transforms/hoisting.cc

```diff
@@ -122,7 +126,7 @@ void HoistingState::doHoisting() {
     // Yes?  Hoist to init, add results as possible cross function results
     innerOp.moveBefore(initPack);
```

Comment:
Additionally, when moving an op, if it uses constants, we need to also copy them into init.